testing/kuttl/e2e/password-change/02-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: password-change-pguser-password-change
+data:
+  # Hardcoding the password as "datalake"
+  password: ZGF0YWxha2U=
```

Comment:
🤔 This looks identical to `02-assert.yaml`. Perhaps this file was supposed to `stringData`?

🤔 Or maybe we don't need the assert?

Reply:
You're right, this assert is incomplete -- I'm just saying "change this and then verify that it changed." What I really need here is some way for the assert to verify that the updated secret has gone through a reconcile (easy enough when I blank out the verifier, but a little harder in the case where I set the verifier... oh, wait, maybe I can just blank out one of the other fields like `uri` and then assert that it's not empty)

Reply:
Updated